msal/consumer-rules.pro

```diff
@@ -17,7 +17,7 @@
 #}
 
 ##---------------Begin: proguard configuration for MSAL  --------
--keep class com.microsoft.** { *; }
+-keep class com.microsoft.identity.client.** { *; }
```

Comment:
```suggestion
-keep class com.microsoft.identity.** { *; }
```

Reply:
applied (other reviewer commented the same)